.github/workflows/test.yml

```diff
@@ -77,6 +77,15 @@ jobs:
         env:
           JULIA_PKG_SERVER: ${{ matrix.pkg-server }}
           JULIA_TEST_VERBOSE_LOGS_DIR: ${{ github.workspace }}
+          JULIA_DEPOT_PATH: ${{ github.workspace }}/do_not_modify_depot
```

Comment:
Will this include the stdlib depot? We don't want to have to recompile all the stdlib packages.